pkg/backup/resource_backupper.go

```diff
@@ -303,8 +303,8 @@ func (rb *defaultResourceBackupper) backupCRD(
 	gr schema.GroupResource,
 	itemBackupper ItemBackupper,
 ) {
-	crdGr := schema.GroupResource{Group: apiextv1beta1.GroupName, Resource: "customresourcedefinitions"}
-	crdClient, err := rb.dynamicFactory.ClientForGroupVersionResource(apiextv1beta1.SchemeGroupVersion,
+	crdGr := schema.GroupResource{Group: apiextv1.GroupName, Resource: "customresourcedefinitions"}
```

Comment:
take a look at what I've done in https://github.com/vmware-tanzu/velero/pull/2230 -- I'm using the discovery helper to determine the server-preferred API version, and backing up that one. I think that's the path we'll want to go down here, rather than hardcoding either `v1beta1` or `v1`.

Reply:
Thanks for the advice.

Reply:
I have reverted the related change in resource_backupper.go and let #2230 address backup issue. This PR address on the install issue only.